utils/fstack.c

```diff
@@ -677,6 +677,9 @@ static int fstack_check_skip(struct uftrace_task_reader *task,
 		return -1;
 
 	if (rstack->type == UFTRACE_EXIT) {
+		if (task->stack_count < 1)
+			return -1;
```

Comment:
Hmm.. it should not have 0 stack count here and be skipped.  Let's return 0 instead.

Reply:
done.